libass/ass_render.c

```diff
@@ -3008,16 +3008,16 @@ fix_collisions(ASS_Renderer *render_priv, EventImages *imgs, int cnt)
                     priv->width = 0;
                 }
             if (priv->height > 0) {     // still a fixed event
-                used[cnt_used].a = priv->top;
-                used[cnt_used].b = priv->top + priv->height;
-                used[cnt_used].ha = priv->left;
-                used[cnt_used].hb = priv->left + priv->width;
+                used[cnt_used].y0 = priv->top;
+                used[cnt_used].y1 = priv->top + priv->height;
+                used[cnt_used].x0 = priv->left;
+                used[cnt_used].x1 = priv->left + priv->width;
                 cnt_used++;
                 shift_event(render_priv, imgs + i, priv->top - imgs[i].top);
             }
         }
     }
-    qsort(used, cnt_used, sizeof(Segment), cmp_segment);
+    qsort(used, cnt_used, sizeof(*used), cmp_rect_y0);
```

Comment:
Minor nit: you replaced `sizeof(type)` by `sizeof(object)` here, but not in the very similar line in `fit_rect`. Seems strangely inconsistent.

Reply:
Thanks, fixed it.